src/ethereum_test/types.py

```diff
@@ -398,22 +597,44 @@ def default(self, obj):
 
             return tx
         elif isinstance(obj, Environment):
-            return {
+            env = {
                 "currentCoinbase": obj.coinbase,
-                "currentDifficulty": hex(obj.difficulty),
-                "parentDifficulty": None,
                 "currentGasLimit": str(obj.gas_limit),
                 "currentNumber": str(obj.number),
                 "currentTimestamp": str(obj.timestamp),
-                "parentTimstamp": None,
-                "blockHashes": {},
+                "currentRandom": str(obj.prev_randao)
```

Comment:
Could these be replaced with a function like `valueOrNone(x)` ?

Reply:
Created a couple of helpers for this.